src/site/content/en/blog/badging-api/index.md

```diff
@@ -126,89 +115,86 @@ and users must add it to their home screens.
 
 // Clear the badge
-navigator.clearExperimentalAppBadge();
+navigator.clearAppBadge();
 ```
 
-`setExperimentalAppBadge()` and `clearExperimentalAppBadge()` can be called from
-a foreground page, or potentially in the future, a service worker. In either
-case, it affects the whole app, not just the current page.
+You can only call `setAppBadge()` and `clearAppBadge()`
+from a foreground page. In the future, you may be able to call it from a service worker. In
+either case, it affects the whole app, not just the current page.
 
 In some cases, the OS may not allow the exact representation of the badge.
 In such cases, the browser will attempt to provide the best representation for
 that device. For example, because the Badging API isn't supported on Android,
 Android only ever shows a dot instead of a numeric value.
 
-Don't assume anything about how the user agent wants to display the badge.
+Don't assume anything about how the user agent displays the badge.
 Some user agents may take a number like "4000" and rewrite it as
 "99+". If you saturate the badge yourself (for example by setting it to "99")
 then the "+" won't appear. No matter the actual number, just call
-`setExperimentalAppBadge(unreadCount)` and let the user agent deal with
+`setAppBadge(unreadCount)` and let the user agent deal with
 displaying it accordingly.
 
 While the Badging API *in Chrome* requires an installed app, you shouldn't
 make calls to the Badging API dependent on the install state. Just call the
 API when it exists, as other browsers may show the badge in other places.
 If it works, it works. If not, it simply doesn't.
 
-## Supporting both the first and second origin trial {: #backwards-compat }
-
-In Chrome 79, a [second origin trial]({{origin_trial.url}}) was started. Based
-on developer feedback received during the first origin trial, the API has
-changed, and is not backwards compatible with the first origin trial.
+## Supporting both the origin trial and the stable API {: #supporting }
 
-If you have an existing implementation, and want to provide backwards
-compatibility for your users, you can wrap the calls in the following
-wrapper function. Once all of your users have migrated to Chrome 79 or later,
-you can remove this wrapper function from your code.
+In the Chrome 79 and Chrome 80 [origin trial]({{origin_trial.url}}) the members
+of the badging API have different names than those in the current implementation. If
+you're participating in that origin trial, you'll need to support both the
+origin trail and the standard names. You can wrap the calls in the following
+functions. Once the origin trial is over or all your users have migrated to
+Chrome 81 or later, you can remove these functions from your code.
 
 ```js
 function setBadge(...args) {
   if (navigator.setExperimentalAppBadge) {
     navigator.setExperimentalAppBadge(...args);
-  } else if (window.ExperimentalBadge) {
-    window.ExperimentalBadge.set(...args);
+  } else {
```

Comment:
Nit: I would prefer to swap these cases around, so the "normal" case is the final API, with a fallback to the experimental API. (That's how it was structured previously, check for the new API first, fall back to the old API.)

So `if (navigator.setAppBadge) { do it } else { old one }`.

Reply:
I don't remember why I did it that way. @petele why did you approve it that way?